accounts/abi/bind/bind_test.go

```diff
@@ -962,11 +975,22 @@ func TestBindings(t *testing.T) {
 			t.Fatalf("test %d: failed to write tests: %v", i, err)
 		}
 	}
+	// Convert the package to go modules and use the current source
+	moder := exec.Command(gocmd, "mod", "init", "bindtest")
+	moder.Dir = pkg
+	if out, err := moder.CombinedOutput(); err != nil {
+		t.Fatalf("failed to convert binding test to modules: %v\n%s", err, out)
+	}
+	pwd, _ := os.Getwd()
+	replacer := exec.Command(gocmd, "mod", "edit", "-replace", "github.com/klaytn/klaytn="+filepath.Join(pwd, "..", "..", "..")) // Repo root
+	replacer.Dir = pkg
+	if out, err := replacer.CombinedOutput(); err != nil {
+		t.Fatalf("failed to replace binding test dependency to current source tree: %v\n%s", err, out)
+	}
```

Comment:
Why are we testing `go mod init` and `go mod edit -replace`?

Reply:
It's not an actual testing part. `TestBindings` generates mini tests and those tests are defined at `bindTests` variable. Those mini test files are named as `xxx_test.go` at the randomly generated folder. We use go module, so it generates mod file by `go mod init` and `go mod edit -replace` file.